Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1177,6 +1177,32 @@ def empty(x):
         return {k: v for k, v in ((k, remove_empty_elements(v)) for k, v in d.items()) if not empty(v)}
 
 
+def remove_none_elements(data):
+    """
+        Remove None values from a dictionary. (updating the given dictionary)
+        Simillar to remove_nulls_from_dictionary but only removes None and not other "empty" elements.
+
+        :type data: ``dict``
+        :param data: The dict to modify (required)
+
+        :return: list of keys removed
+        :rtype: ``list``
+    """
+    list_of_keys = list(data.keys())
+    res = []
+    for key in list_of_keys:
+        if data[key] is None:
+            res.append(key)
+            del data[key]
+    return res
+
+
+if (not os.getenv('COMMON_SERVER_NO_AUTO_PARAMS_REMOVE_NULLS')) and hasattr(demisto, 'params'):
+    res = remove_none_elements(demisto.params())
```

Comment:
this can break BC, no?
If the integration has code like `demisto.params()['status']`. So instead of returning None, it will fail on KeyError.

Reply:
Will try a different approach.

Reply:
New approach. See update